### ISSUE TITLE
push our i8n upstream to BC's devex repo

### ISSUE THREAD

**Issue by RachelMuston** [canada-ca/devex#7]
includes all changes we made to basic functionality to include French as well as English.

**Comment by markwooff** [comment]
Yes, I'm hoping to make the i8n easier to manage before pushing back upstream. This would allow others to add even more languages easily going forward 👍
